zinc/src/sbt-test/source-dependencies/sealed/test

```diff
@@ -5,5 +5,4 @@ $ copy-file changes/A.scala A.scala
 
 # D.scala needs recompiling because the pattern match in D
 # is no longer exhaustive, which emits a warning
-> checkRecompilations 1
-> checkRecompilations 2 A B C D E
+-> compile
```

Comment:
The sealed test existed since 2009 (https://github.com/sbt/sbt/blob/5b1e26660c9bb329b7b35adb3fe61e663508a264/src/sbt-test/source-dependencies/sealed/test), and with `-Xfatal-warnings` in 2011 (https://github.com/sbt/sbt/commit/ff95799af710cb19286a961ec01d1bbeffa4b1d9#diff-499c993bb84e4d6d000b053093ad9f8b). I am guessing that `checkRecompilations` was a workaround prior to `incOptions.properties` support.